keyboards/rastersoft/minitkl/info.json

```diff
@@ -0,0 +1,336 @@
+{
+    "keyboard_name": "MiniTKL",
+    "manufacturer": "Rastersoft",
+    "url": "https://gitlab.com/rastersoft/full-ten-keyless",
+    "maintainer": "rastersoft",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x68F1",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["D6", "D4", "B6", "D5", "B4", "D3", "D7", "D1", "C6", "D2", "C7", "D0"],
+        "cols": ["F4", "F1", "F0", "F5", "F6", "F7", "B7", "B5"]
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "rgblight": true
+    },
+    "rgblight": {
+        "led_count": 3,
+        "pin": "B2",
```

Comment:
This will need to be updated on develop.

Reply:
Do you mean that I should rebase with develop instead of main? I presumed that it was only for changes in the code, not for new keyboards...